crowdin.yml

```diff
@@ -1,28 +1,11 @@
 project_id_env: _CROWDIN_PROJECT_ID
 api_token_env: CROWDIN_API_TOKEN
-preserve_hierarchy: true
```

Comment:
`preserve_hierarchy: true` is required when `dest` is used in source configs below.